staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1019,6 +1019,8 @@ message WindowsContainerResources {
     int64 memory_limit_in_bytes = 4;
     // Specifies the size of the rootfs / scratch space in bytes to be configured for this container. Default: 0 (not specified).
     int64 rootfs_size_in_bytes = 5;
+    // Specifies the set of CPUs to affinitize for this container.
+    repeated uint64 affinity_cpus = 6;
```

Comment:
I believe we are going to need a structure like https://learn.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-group_affinity, so we can support machines with groups > 64 processors